apps/src/templates/congratsBeyondHocActivityCards.js

```diff
@@ -20,7 +20,7 @@ const CSFAccelerated = {
 const CourseCatalog = {
   title: i18n.courseCatalogTitle(),
   description: i18n.courseCatalogDescription(),
-  link: "/courses/view?=teacher",
```

Comment:
This should have been `/courses?view=teacher` fwiw. Was the intent to show the teacher version of /courses?  if you're clicking on this when signed out and want a teacher view, you need this parameter.

Reply:
For non-EN it doesn't matter (they get the same). So I prefer the simple `/courses` without the teacher param.